src/styles/contacts.scss

```diff
@@ -0,0 +1,135 @@
+.contact {
+  display: grid;
+
+  padding-top: 48px;
+
+  &__user {
+    grid-column: 1 / 6;
+
+    display: flex;
+    gap: 16px;
+    flex-direction: column;
+  }
+
+  &__info {
+    box-sizing: border-box;
+
+    padding-left: 16px;
+    height: 48px;
+
+    background: #f8f8fa;
+    border: 1px solid transparent;
+    border-radius: 40px;
+    outline: none;
+
+    &::placeholder {
+      font-family: "Inter", sans-serif;
+      font-size: 14px;
+      line-height: 20px;
+    }
+
+    &:nth-child(3) {
+      padding-top: 14px;
+
+      border-radius: 16px;
+      padding-bottom: 148px;
+      overflow: hidden;
+      resize: none;
+    }
+
+    &:hover {
+      border: 1px solid #d5d7de;
+    }
+
+    &:focus {
+      border: 1px solid #292929;
+    }
+  }
+
+  &__button {
+    margin-top: 16px;
+
+    height: 56px;
+
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 27px;
+
+    border: 3px solid #292929;
```

Comment:
In this case better use `transparent` border color, because your `background-color` the same.